src/iuse.cpp

```diff
@@ -1841,6 +1841,32 @@ int iuse::purify_iv(player *p, item *it, bool, const tripoint& )
     return it->type->charges_to_use();
 }
 
+void spawn_spores( const player &p ) {
+    int spores_spawned = 0;
+    for ( const tripoint &dest : closest_tripoints_first( 4, p.pos() ) ) {
```

Comment:
changed from points_in_radius to closest_tripoints_first so that monster spawning locus will be centered on the player rather than northwest of the player.